src/System.Slices/src/System/Span.cs

```diff
@@ -131,6 +131,12 @@ internal Span(object obj, UIntPtr offset, int length)
             Length = length;
         }
 
+        public void Clear()
```

Comment:
Now that I think about it, this change isn't necessary since I can just set the callers reference to default(T) to clear it. Will revert